src/open_inwoner/templates/pages/search.html

```diff
@@ -1,5 +1,5 @@
 {% extends 'master.html' %}
-{% load i18n form_tags utils icon_tags grid_tags pagination_tags filter_tags %}
+{% load i18n form_tags string_tags icon_tags grid_tags pagination_tags filter_tags %}
```

Comment:
```suggestion
{% load i18n form_tags icon_tags grid_tags pagination_tags filter_tags %}
```

String_tags is included, but `safe` is part of the built-in filters and can be used without this inclusion.

Reply:
Ah yes - in my first version I made use of our `optional_paragraph` tag which is part of `string_tags`. I'll remove it here.